### PR TITLE
space-time none coordinate variable should not be checked in

### PR DESCRIPTION
**Pull request by fxmzb123**
check_independent_axis_dimensions.
The check has been changed to check:
1) whether variable dimension is a space-time dimension
2) whether variable dimension has associated coordinated variable.
Issue 72

### CODE REVIEW

File: compliance_checker/cf/cf.py [hunk 1]
```diff
@@ -1340,7 +1346,7 @@ def check_independent_axis_dimensions(self, ds):
                 space_time_coord_var.append(each._name)
 
         #Find all all space-time variables that are not coordinate variables
-        space_time_non_coord_var=[]
+        ''' space_time_non_coord_var=[]
```
Comment (daf) [inline compliance_checker/cf/cf.py]: Instead of commenting the block, just delete it.

**Comment by daf** [comment]
Hi @fxmzb123, thanks for the patch.  There is a test failure (`NameError: global name 'dim_name' is not defined`) which is the the invalid handler path - you had removed the dim_name setting line from the else branch.  You can use `each` in the error message in place of it, but I think that will be invalid if we move the checker to Python 3.

Then, after fixing this error locally, the test still fails as I think it is depending on the old logic.  Can you look at the test and see if it needs to be updated too?  Thanks.

**Comment by fxmzb123** [comment]
Hi Dave,

Thank you for the comment.

I have removed not used code, fixed the missing dim_name variable and changed related test. I have also changed the check logic a little bit as the logic was not quite right.

Cheers,
Ming

**Comment by lukecampbell** [comment]
Is this ready to go?

**Comment by lukecampbell** [comment]
@daf

**Comment by daf** [comment]
@lukecampbell will review today

**Pull request merged**